tests/flow/test_udf.py

```
@@ -529,6 +529,64 @@ def sort_entities(entities):
+        expected = [bob_knows_alice]
+
+        actual   = sort_entities(actual)
+        expected = sort_entities(expected)
+        self.env.assertEqual(actual, expected)
+
+        # 14 Alice incoming edges
+        # all possible relationship types except 'KNOWS'
+        types = ['WORKS_AT', 'LIVES_IN', 'VISITED', 'USES']
+        for t in types:
+            # make sure to include the 'KNOWS' relationship type
+            t = ['KNOWS'] + [t]
```

Ruff (0.14.11)

Consider `['KNOWS', t]` instead of concatenation

Replace with `['KNOWS', t]`

(RUF005)